metaflow/cli.py

```diff
@@ -837,6 +859,7 @@ def version(obj):
     multiple=True,
     help="Add a decorator to all steps. You can specify this option "
     "multiple times to attach multiple decorators in steps.",
+    callback=config_merge_cb,
```

Comment:
also need the callback for the `run` subcommand I guess? 
right now this works
```
METAFLOW_DECOSPECS="kubernetes:memory=512" python HelloFlow.py --with "kubernetes:memory=4096,cpu=2" run
```
but this does not
```
METAFLOW_DECOSPECS="kubernetes:memory=512" python HelloFlow.py run --with "kubernetes:memory=4096,cpu=2"
```

Reply:
fixed with latest commit :)